docs/core/platform/significant_change.md

```diff
@@ -8,17 +8,18 @@ This support is added by creating a `significant_change.py` platform file with a
 
 ```python
 from typing import Any, Optional
-from homeassistant.core import callback
+from homeassistant.core import HomeAssistant, callback
 
+# pylint: disable=unused-argument
```

Comment:
Disabling linter rules in code is bad practice, let's not advertise that.

Reply:
Removed pylint rule.  But is there a way to not accept those args which are not used?   In my case a significant change is any primary change, not an attribute change.   I am not using kwargs or the attrib states.

Reply:
Use an `_` as the var name.